userguide/storage.rst

```diff
@@ -279,6 +265,29 @@ A pop-up message shows a reminder that
 the key, the data on the disks is inaccessible. Refer to
 :ref:`Managing Encrypted Volumes` for instructions.
 
+.. _Encryption performance:
+
+Encryption performance
+^^^^^^^^^^^^^^^^^^^^^^
+
+If the processor supports the 
+`AES-NI <https://en.wikipedia.org/wiki/AES-NI#Supporting_CPUs>`_
+instruction set, there is very little, if any, degradation in
+performance when using encryption and only a few disks.
+Performance will suffer if the CPU does not support
+AES-NI or if no crypto hardware is installed.  Without hardware
+acceleration, there will be about a 20% performance decrease for a
```

Comment:
I'm hesitant to give a flat 20% number here, since it is likely to be dependent on more than one factor.

Reply:
It was in the original text so I left it. Let me know any preferred wording.  But overall my gut feeling is that it's better to give some "ballpark" indication (1%? 5%? 70%?) than none at all, as many just want "some idea" whether it's a showstopper or not.   Users who need more certainty will probably benchmark.